extensions/ql-vscode/README.md

```diff
@@ -110,3 +110,7 @@ For more information about the CodeQL extension, [see the documentation](https:/
 ## License
 
 The CodeQL extension for Visual Studio Code is [licensed](LICENSE.md) under the MIT License. The version of CodeQL used by the CodeQL extension is subject to the [GitHub CodeQL Terms & Conditions](https://securitylab.github.com/tools/codeql/license).
+
+## Data and Telemetry
+
+If you specifically opt-in to permit GitHub to do so, GitHub will collect usage data and metrics for the purposes of helping the core developers to improve the CodeQL extension for VS Code. This data will not be shared with any parties outside of GitHub and will be retained for a maximum of 30 days. Please see [telemetry](TELEMENTRY.md) for more information.
```

Comment:
As per my other comments, it'd be great if we could work in the word "anonymized" a few times :slightly_smiling_face: